pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -219,6 +194,10 @@ func (r *Reconciler) ensureTargetNamespaceExists(ctx context.Context, tc *v1alph
 			}
 		}
 	} else {
+		labels := map[string]string{"operator.tekton.dev/targetNamespace": "true"}
```

Comment:
I can see that the following label already exists at line https://github.com/tektoncd/operator/pull/600/files#diff-de56e5fd636d3b9b5ab99eb6197436045c5bc27187a3f59649bae7d66a8e7202R19, so do we still need to pass this label from here?

Reply:
So from the tektonConfig I have removed but in the tektonPipelines we need it

Reply:
why is it not needed in tektonConfig. I was thinking we will need it in both TektonConfig and TektonPipeline.

Reply:
It is needed in tektonConfig as well, but the reason I've remove this variable because, in the `createTargetNamespace` function itself I'm adding that label while creating the namespace. For ref: https://github.com/tektoncd/operator/pull/600/files#diff-de56e5fd636d3b9b5ab99eb6197436045c5bc27187a3f59649bae7d66a8e7202R20